modules/katamari/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+- Added a `Repeatable` type to the `Singleton` module
```

Comment:
```suggestion
- Added a `Repeatable` API to the `Singleton` module to repeat a function at a fixed interval.
```
Maybe? Just that it's not a `type` so that's misleading.